cobbler/cobbler_collections/collection.py

```diff
@@ -420,6 +422,7 @@ def add(
         with_sync: bool = True,
         quick_pxe_update: bool = False,
         check_for_duplicate_names: bool = False,
+        rebuild_menu: bool = True,
```

Comment:
Why not use the `with_sync` parameter?

Reply:
Because sync gets the particular item as an argument, so it is not possible to postpone the call at the end of transaction (at least without significant rewrite).

Rebuild menu is the operation which is slow and it can be postponed without problems.